app/views/listings/_edit.html.erb

```diff
@@ -0,0 +1,129 @@
+<div id="editListingModal" role="dialog" tabindex="-1" class="modal fade text-left show" style="margin-top: 20px;">
```

Comment:
- Listing attributes have their own pages and forms.
- Create a method for each set of attributes and add them as member routes to `resources :listings`.
Based on Sample Layout (Please check the wireframe for more info on each page):
<img width="1792" alt="Screen Shot 2021-03-22 at 6 39 19 PM" src="https://user-images.githubusercontent.com/20994464/111977631-ed928300-8b3d-11eb-840f-56a05b4d257c.png">